deps/Makefile

```diff
@@ -53,6 +54,12 @@ linenoise: .make-prerequisites
 
 .PHONY: linenoise
 
+nvml: .make-prerequisites
+	@printf '%b %b\n' $(MAKECOLOR)MAKE$(ENDCOLOR) $(BINCOLOR)$@$(ENDCOLOR)
+	cd nvml && $(MAKE)
```

Comment:
This will build both debug and nondebug version of the library, but we're using only one variant. Maybe we could look at $OPTIMIZATION variable and decide which variant to use?